source/_integrations/sunsynkweb.markdown

```diff
@@ -0,0 +1,62 @@
+ha_integration_type: integration
+
+---
+
+The **Sunsynk web** {% term integration %} gathers statistics from [Sunsynk connect](http://sunsynk.org). It exposes real-time and historical information.
+
+This will log into your Sunsynk account and will collate the information of any plants or inverters and present the main sensors for the plant, real-time and historical.
+
+## Prerequisites
+
+To use this integration, you will need a sunsynk-branded e-linter [wifi dongle](https://www.sunsynk.org/logger-support) for your inverter(s) and a [Sunsynk connect](Sunsynk.net) account. This will typically be created by your installer on your behalf. The account needs at least the 'visitor' permissions.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The integration exposes the following sensors:
+
+If you have multiple plants (inverters), the sensors represent an aggregated
+view of the relevant sensors. For example: in a typical installation, the battery state of charge is the maximum battery charge reported by both inverters. This is because the batteries are on a shared bus. 
+The load power will be the sum of the load on each inverter.
+The energy totals produce statistics that are usable with the energy dashboard.
+
+| ID                     | Description                                               |
+| ----------------------- | --------------------------------------------------------- |
+| `battery_power`         |  Solar supply battery power                               |
+| `load_power`            |  Total inverter load                                      |  
+| `grid_power`            |  Load to or from the grid instantaneous                   |  
+| `pv_power`              |  Power generated by solar panels instantaneous            |  
+| `state_of_charge`       |  Solar battery charge                                     |  
+| `acc_pv`                |  Total PV generated energy                                |  
+| `acc_load`              |  Total energy consumed                                    |  
+| `acc_grid_import`       |  Total energy imported from the grid                      |  
+| `acc_grid_export`       |  Total energy exported to the grid                        |  
+| `acc_battery_charge`    |  Accumulated energy charged into the batteries            |  
+| `acc_battery_discharge` |  Accumulated energy discharged from the batteries         |
+
+## Other considerations
+
+The Sunsynk web API and website seem to be rebranded [e-linter](https://www.e-linter.com) services.
+Currently, the underlying wrapper [pysunsybkweb](https://github.com/francoisverbeek/pysunsynkweb) has hardcoded
+references to Sunsynk. If you are using another brand that uses the same underlying service, get in touch, we may
+be able to extend the support to additional tenants of e-linter's infrastructure.
+
+## Disclaimer
+
+This work is done without Sunsynk or E-linter's involvement or endorsement, based on public documentation.
```

Comment:
not sure about the spelling of e-linter. I see the variant E-linter and e-liner on this page. On their own website, the version I saw was E-LINTER.